modules/chinesewhispers/src/main/java/org/clueminer/chinesewhispers/ChineseWhispers.java

```diff
@@ -1,115 +1,115 @@
-            changes = false;
-            for (Node nodeIt : nodes) {
-                AdjListNode node = (AdjListNode) nodeIt;
-                HashMap<Long, Integer> classes = new HashMap<>(nodes.size());
-                Long maxClass = node.getId();
-                Integer maxCount = 1;
-                for (Node neighborIt : graph.getNeighbors(node)) {
-                    AdjListNode neighbor = (AdjListNode) neighborIt;
-                    Long classValue = (Long) neighbor.getInstance().classValue();
-                    Integer count = classes.get(classValue);
-                    count = count == null ? 1 : count + 1;
-                    classes.put(classValue, count);
-                    if (count > maxCount
-                            || (count.equals(maxCount) && random.nextBoolean())) {
-                        maxCount = count;
-                        maxClass = classValue;
-                    }
-                }
-                if (!maxClass.equals(node.getInstance().classValue())) {
-                    changes = true;
-                    node.getInstance().setClassValue(maxClass);
-                }
-            }
-        }
-
-        //graph.print();
-        HashMap<Long, List<Node>> clusters = new HashMap<>();
-        for (Node node : graph.getNodes()) {
-            Long classValue = (Long) node.getInstance().classValue();
-            List<Node> thisCluster = clusters.get(classValue);
-            if (thisCluster == null) {
-                thisCluster = new ArrayList<>();
-                clusters.put(classValue, thisCluster);
-            }
-            thisCluster.add(node);
-        }
-        Clustering result = Clusterings.newList();
-        for (Map.Entry<Long, List<Node>> entrySet : clusters.entrySet()) {
-            //Long clusterId = entrySet.getKey();
-            List<Node> clusterNodes = entrySet.getValue();
-            Cluster cluster = result.createCluster();
-            //System.out.println("Cluster " + clusterId);
-            for (Node clusterNode : clusterNodes) {
-                //System.out.println("\t" + clusterNode.getId());
-                cluster.add(clusterNode.getInstance());
-            }
-            //System.out.println("");
-        }
-        return result;
-    }
-
-    private void createEdges(Dataset<? extends Instance> dataset, List<Node> nodes) {
-        for (int instanceIdx = 0; instanceIdx < dataset.size(); instanceIdx++) {
-            for (int attributeIdx = 0; attributeIdx < dataset.attributeCount(); attributeIdx++) {
-                if (dataset.get(instanceIdx, attributeIdx) > 0.5) {
-                    AdjListNode source = (AdjListNode) nodes.get(instanceIdx);
-                    AdjListNode target = (AdjListNode) nodes.get(attributeIdx);
-                    Edge edge = graph.getFactory().newEdge(source, target);
-                    graph.addEdge(edge);
-                }
-            }
-        }
-    }
-}
+package org.clueminer.chinesewhispers;

```

Comment:
the diff doesn't work due to different line endings, could you please change it to Unix LF? (instead of CRLF)